ipod-shuffle-4g.py

```diff
@@ -500,6 +523,8 @@ def populate_directory(self, playlistpath, recursive = True):
         # Folders containing no music and only a single Album
         # would generate duplicated playlists. That is intended and "wont fix".
         # Empty folders (inside the music path) will generate an error -> "wont fix".
+        if "/iPod_Control/Podcasts/" in playlistpath:
+            self.set_podcast()
```

Comment:
Dont't we need to move that down, so we can also recognize m3u and pls playlists as podcast playlists?

Reply:
I'm not sure where you want to move it down to. Do you mean outside of the `populate_directory` method and just after the if/else branch the call to `populate_directory` is in?

Reply:
inside the general populate function